packages/browser-tests/.env

```diff
@@ -0,0 +1,3 @@
+EXAMPLE_TEMPLATE="web-chat"
```

Comment:
I wonder what is the best practice here as I assume this will fail if we do any breaking change in the API.

Reply:
Once breaking change is introduced to an example under testing it won't affect CI right away since examples are shipped with a particular version of `@waku/create-app`. 

Or what API do you mean?

Reply:
> Once breaking change is introduced to an example under testing it won't affect CI right away since examples are shipped with a particular version of @waku/create-app.

From the PR's description:

> it injects links to repository's version of packages;

what _repository_ are we referring to? My understanding is:

1. Pull latest web-chat code from js-waku-examples repo
2. Does some symlink to replace web-chat's dependency with code from the current js-waku git branch
3. build web-chat, runs some test

if a breaking change on any of the js-waku package happens, then the web-chat code pull cannot use the new changed js-waku API, and hence the CI would fail.

What part am I not understanding?

Reply:
you are right, to solve this one should update an example to correspond to the breaking changes in `js-waku`
To do this there are two ways:
- [ ] locally bind to the latest version of `js-waku` and push changes to an example - dirty and doesn't really cover everything;
- [ ] some intermediate publications of `js-waku` so this issue should solve it - https://github.com/waku-org/js-waku/issues/1543

Reply:
that means the flow for release will be something like:
- check if test against example fails;
- if yes - go and upgrade example to `next` package version;
- check again - if ok - release;